modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -554,14 +547,12 @@ export default function EditNotificationTemplate({
 									value={values.description}
 								/>
 
-								{!Liferay.FeatureFlags['LPS-162133'] && (
-									<SingleSelect
-										disabled
-										label={Liferay.Language.get('type')}
-										options={[]}
-										value={Liferay.Language.get('email')}
-									/>
-								)}
+								<SingleSelect
```

Comment:
@p-albuquerque you need to remove this SingleSelect. It was only meant to be displayed while the Feature Flag was disabled.